client/src/App.jsx

```diff
@@ -1,35 +1,34 @@
 import { useState, useEffect } from 'react';
+//'switch' has been replaced with 'routes'
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+import CreateRoseBudThorn from './components/CreateRoseBudThorn';
+import HomePage from './components/HomePage';
+import ListLatestEntrie from './components/ListLatestEntries';
```

Comment:
```suggestion
import ListLatestEntries from './components/ListLatestEntries';
```
this may mean you need to update your return statement where you are using this component

Reply:
Thank you!  Good catch!